iOS/Flows/Conversation/_New/ConversationViewController.swift

```diff
@@ -34,15 +33,14 @@ class ConversationViewController: FullScreenViewController,
     override var inputAccessoryView: UIView? {
         return self.messageInputAccessoryView
     }
-    
     override var canBecomeFirstResponder: Bool {
         return true
     }
     
     init(conversation: Conversation?) {
         if let conversation = conversation {
             self.conversationController
-            = ChatClient.shared.channelController(for: conversation.cid, messageOrdering: .bottomToTop)
+            = ChatClient.shared.channelController(for: conversation.cid, messageOrdering: .topToBottom)
```

Comment:
Because the cells are right to left, we can load the messages in the default way.